docs/Networking.asciidoc

```diff
@@ -283,6 +287,11 @@ DHCLIENT_SET_DEFAULT_ROUTE='yes'
 
 * Start _wicked_ as network service:
 
+Pros of wicked over NetworkManager:
+- +Proper IPv6 support
```

Comment:
Use regular bullet points. When I use "Display rich text diff" it seems that these points confuse the parser.